tg_bot/__main__.py

```diff
@@ -415,6 +415,34 @@ def migrate_chats(bot: Bot, update: Update):
     LOGGER.info("Successfully migrated!")
     raise DispatcherHandlerStop
 
+def is_chat_allowed(bot, update):
```

Comment:
This should raise dispatcher handler stop if the bot leaves, to avoid further updates being handled.